### PR TITLE
Updated .net framwork to 4.6.1, removed unused directives and updated…

### PR DESCRIPTION
**Pull request by cjames9001**
… nunit to be a nuget package instead of a reference to my local machine's version -Oh how much I've learned since 2012

### CODE REVIEW

File: MovieQueueManagerGUI/MovieQueueManagerGUI.csproj [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (taylorglenn) [inline MovieQueueManagerGUI/MovieQueueManagerGUI.csproj:2]: So here you update from 4 to 12, but what exactly is it that you're updating?  What's a schema? When I go to the URL, it just says that the resource has been moved.

File: QueueManager/QueueFileManager.cs [hunk 2]
```diff
@@ -1,7 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
```
Comment (taylorglenn) [inline QueueManager/QueueFileManager.cs:4]: did you ever use these?  or were they just put there by visual studio?

Comment (cjames9001) [inline QueueManager/QueueFileManager.cs:4]: When you create a new class in VS it puts in a few defaults. It's only in recent versions of VS that they grey them out to let you know when they aren't used.

File: MovieQueueManagerGUI/MovieQueueManagerGUI.csproj (hunk 1 above)
Comment (cjames9001) [inline MovieQueueManagerGUI/MovieQueueManagerGUI.csproj:2]: Almost anything that happens in *.csproj files is auto generated. When I upgraded this project to .net 4.6.1 it also updated the version of MSBuild that it uses to build the project. The schema is referring to what elements are allowed/expected in an XML file. These two links may help give you some background: [general info about xmlns](http://www.w3schools.com/xml/xml_namespaces.asp) and [specifically how Microsoft uses it](https://msdn.microsoft.com/en-us/library/ms754619%28v=vs.85%29.aspx)

**Comment by cjames9001** [comment]
@taylorglenn next time you can just comment once you're done with any questions or comments that you approve or that its been reviewed. That way I'll know without us having to see each other and chat about it.

**Pull request merged**